com.netxms.NetXMSClient50.yaml

```diff
@@ -41,7 +41,7 @@ modules:
         sha256: 928e1f11ef1d93592295d1038dbbe61f647b839d318235112123dbbe73a512d6
       - type: file
         url: https://raw.githubusercontent.com/netxms/appstream/master/5.0/com.netxms.NetXMSClient50.metainfo.xml
```

Comment:
you should NEVER reference a branch. This will break the build. You just proved it.